# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/imakemdep_ppc.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config/imake/imakemdep.h.orig	Wed Nov 29 17:33:50 2000
+++ ./config/imake/imakemdep.h	Thu Dec 28 15:44:10 2000
@@ -364,6 +364,9 @@
 # ifdef __m68k__
 	"-D__m68k__",
 # endif
+# ifdef __powerpc__
+	"-D__powerpc__",
+# endif
 # ifdef __GNUC__
 	"-traditional",
 # endif
